### DISCUSSION TITLE
From `<style id="stitches" .../>` for SSR, `id="stitches"` is necessary?

### DISCUSSION THREAD

**Discussion by junheep-dev** [stitchesjs/stitches#880] [Help] [answered]
The document says that we need to use a style tag for SSR and it has to have `id="stitches"` attribute.
https://stitches.dev/docs/server-side-rendering
```
<style id="stitches" dangerouslySetInnerHTML={{ __html: getCssText() }} />
```

However, I think this id attribute is not necessary because stitches core does not use this id for finding style sheet while hydration.
Did I something misunderstood? or, we can remove id from the document.

**Comment by peduarte** [comment, marked as answer]
It's not required, but its good practice so you know which style were added by stitches.

**Reply by junheep-dev** [reply]
Okay, thanks for your explanation
